### DISCUSSION TITLE
Doks in Bengali Language, Changing content of my-doks-site\layouts\index.html file

### DISCUSSION THREAD

**Discussion by raqueeb** [thuliteio/doks#101] [Q&A]
Good documentation site. I wanted to change the E:\my-doks-site\config\_default\params.toml to see the reflected change, it only changes the title. npm run start functions properly. 
title = "উৎকর্ষের সন্ধানে"

Rest of the items remain unchanged. How do I change the titleAddition and description?

```
## Homepage
title = "উৎকর্ষের সন্ধানে"
titleSeparator = "-"
titleAddition = "এক জীবনে অনেক কিছু করার চেষ্টা"
description = "আমরা হয়তোবা বাঁচি ৭০-৮০ বছর, সেই তুলনায় আমারা যেতে পারতাম আরো দুরে। সেই দক্ষতা এবং উৎকর্ষের ধারণা নিয়ে আমার গল্প।"
```

It stays with the "Modern Documentation Theme".

If I need to change the content like "Security aware", do I need to change the my-doks-site\layouts\index.html or do you have a markdown file separately?

**Comment by epogrebnyak** [comment]
Several suggestions: 

- you need to change both content/_index.md and layout.html to change the landing page
- consider changing default language in config.toml 
- see #55 for upcoming multilang example

**Reply by raqueeb** [reply]
Thank you. 

```
titleAddition = "...."
description = "....."
```

These don't get changed. How do I change these in the picture?

![image](https://user-images.githubusercontent.com/32799073/106634305-dd344200-65a9-11eb-8a23-b60576a7e9eb.png)

**Reply by h-enk** [reply]
In `./layouts/index.html` on line 21

**Reply by raqueeb** [reply]
Thank you. It means I need to edit that html file. I assume it was coming from a markdown file.

**Reply by epogrebnyak** [reply]
Just to shed some more light inside https://github.com/h-enk/doks/blob/master/layouts/index.html there are three tags that come from markdown or global project settings: `.Title`, ` .Params.lead` (markdown _index.md) and `.Site.Data.doks.version` (global config) and the rest is in hardcoded html. Hugo works by applying a layout to markdown files and using the variables from markdown front matter and configuration files. 

Theoretically, there is a possibility to define the text of front page in a separate config option, but that would overcomlicate things, so one can just edit the index.html template.

**Comment by epogrebnyak** [comment]
Should here be `.Params.lead` not `.Params.Lead` - attn @h-enk

https://github.com/h-enk/doks/blob/4e65d012c5887ff8db9650006d8c6f2303d1b338/layouts/index.html#L8

**Reply by h-enk** [reply]
Yes, to keep it consistent. Changed: https://github.com/h-enk/doks/commit/92833b5cfbbb1926562418352fecbb6eb0cc2e18

**Reply by epogrebnyak** [reply]
So quick! )

**Comment by epogrebnyak** [comment]
@raqueeb - I think you need to `title="..."` to `config.toml` 
@h-enk - this is one of things I needed to change while making a translation: it looks one needs `title` in `config.toml`, not `params.toml`.

This works (placing same code in `params.toml` doesn't):
https://gitlab.com/epogrebnyak/doks-howto/-/blob/master/config/_default/config.toml#L13-27

**Reply by raqueeb** [reply]
Thank you.
